packages/apollos-data-prayer/src/prayer-requests/schema.js

```diff
@@ -19,6 +20,8 @@ const prayerRequestSchema = gql`
     deletePublicPrayerRequest(id: String!): PrayerRequest
     incrementPrayed(id: String!): PrayerRequest
     flagRequest(id: String!): PrayerRequest
+    savePrayer(nodeId: String!): PrayerRequest
```

Comment:
```suggestion
    savePrayer(id: String!): PrayerRequest
```

to match others

Reply:
I like the name here. A nodeId and different from a regular id.

Reply:
a `PrayerRequest` extends Node too, are they not the same thing? Like couldn't we also call the variable for all the prayer queries `nodeId`? Or is the Node for the saved stuff not the same kind of node?

Reply:
nodeId is that hashed id we created within the nod apollos framework. Id can mean a number of things.

Reply:
so maybe we rename all the other variables to `nodeID` in the other queries for consistency?